hi-ml-histopathology/src/histopathology/datamodules/base_module.py

```diff
@@ -121,9 +126,15 @@ def _load_dataset(self, tiles_dataset: TilesDataset, stage: str, shuffle: bool)
                 return torch.load(f, map_location=memory_location)
 
         generator = _create_generator(self.seed)
+
+        if stage in ['val', 'test']:
+            eff_max_bag_size = self.max_bag_size_inf
```

Comment:
For the future it might be convenient to decouple also validation and test phase. Validate on the full slide each time might be time consuming, this is why it might be advantageous to have the option to do full slide only for the testing phase.

Reply:
Thanks, we could decouple this in future.